packages/cactus-test-tooling/src/main/typescript/socketio-test-setup-helpers/socketio-test-setup-helpers.ts

```diff
@@ -72,7 +72,7 @@ export function connectTestClient(socket: ClientSocket): Promise<ClientSocket> {
     socket.on("connect_timeout", errorHandlerFactory("connect_timeout"));
 
     socket.on("connect", () => {
-      socket.removeAllListeners();
+      //socket.removeAllListeners();
```

Comment:
@RafaelAPB Please bug me until I come up with an explanation to why this is not compiling.
BTW looking at the code in a little more detail, I'd suspect that maybe the problems with your package dependencies is the culprit. You are doing cross-package imports in the Typescript code, but not declaring the dependencies in the package.json files which is defintely a problem and it could lead to weird compilation errors like this one.